docs/source/contribute.rst

```diff
@@ -0,0 +1,76 @@
+Contribute
+==========
+
+To contribute to the package, please follow the procedures below.
```

Comment:
@XingerTang I propose we show the full work-flow including forking, cloning, ... all the steps starting from having nothing to creating a pull request

Reply:
@gregorgorjanc Sure! I will work on it later.